dataplane/common/src/lib.rs

```diff
@@ -40,3 +40,39 @@ pub struct BackendList {
 
 #[cfg(feature = "user")]
 unsafe impl aya::Pod for BackendList {}
+
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+pub struct ClientKey {
+    pub ip: u32,
+    pub port: u32,
+}
+
+#[cfg(feature = "user")]
+unsafe impl aya::Pod for ClientKey {}
+
+#[derive(Copy, Clone, Debug, Default)]
+#[repr(C)]
+pub enum TCPState {
```

Comment:
So this isn't fully descriptive right? We're using map hueristics to determine weather or not the connection is "new" rather than tracking packets through "Start", "Syn_Sent", "Syn_Rcvd" etc right? 

I if ^^ is the case we need some comments as such on this.  Since when I see `TCPState` it makes me think it should contain all possibel states for a TCP connection